Error when installing the package via pip

When installing your package, the following error is thrown
```
#warning "Using deprecated NumPy API, disable it with " \
     ^
    pandas/_libs/src/ujson/python/objToJSON.c:194:12: error: use of undeclared identifier 'NUMPY_IMPORT_ARRAY_RETVAL'
        return NUMPY_IMPORT_ARRAY_RETVAL;
               ^
    1 warning and 1 error generated.
    error: command 'clang' failed with exit status 1
```

--- a/setup.py
+++ b/setup.py
@@ -65,7 +65,7 @@ setuptools.setup(
     package_dir={'': 'src'},
     packages=setuptools.find_packages('src'),
     install_requires=[
-        'pandas~=0.24.0',
+        'pandas>=0.24.0,<1.0.0',
         'nltk>=3.4.0',
         'textblob>=0.15.3',
         'scikit-learn>=0.21.3',

